sdk/documentintelligence/Azure.AI.DocumentIntelligence/tests/samples/DocumentIntelligenceSamples.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.AI.DocumentIntelligence.Tests;
+using Azure.Core.TestFramework;
+
+namespace Azure.AI.DocumentIntelligence.Samples
+{
+    [LiveOnly]
+    [AsyncOnly] // Ensure that each sample will only run once.
+    public partial class DocumentIntelligenceSamples : RecordedTestBase<DocumentIntelligenceTestEnvironment>
```

Comment:
Ideally it should. We started using `RecordedTestBase` in the previous Form Recognizer package because it was required to make use of the `IgnoreServiceError` attribute. Once we have live tests enabled I may change those back if we see no problems in the pipeline.